https://issues.redhat.com/browse/ACM-23917 MCOA (#8364)
open-cluster-management/rhacm-docs: observability/obs_mcoa_relabel.adoc

[#relabeling-mcoa-default-metrics]
= Relabeling default metrics for the multicluster observability add-on

> Relabel configurations to modify the metrics that you are collecting. Relabel configurations in each `ScrapeConfig` resource or globally in the `remoteWrite` configuration of the `PrometheusAgent` resource.

Reviewer: ```suggestion
Relabel configurations to modify the default metrics that you are collecting. Relabel configurations in each `ScrapeConfig` resource or globally in the `remoteWrite` configuration of the `PrometheusAgent` resource. This can be used to remove some metrics that are collected by default for plaftform monitoring.
```

*Required access:* Cluster administrator

.Prerequisites

- You have installed and enabled the multicluster observability add-on. For more information, see xref:../observability/obs_enable_mcoa.adoc#enable-mcoa[Enabling the multicluster observability add-on].

> Complete the following steps to relabel default metrics for the multicluster observability add-on:

Reviewer: ```suggestion
Complete the following steps to remove a metric collected by default, using relabelling:
```

> . Remove the `Watchdog` alert metric by adding the `metricRelabelings` in the `platform-metrics-alerts` `ScrapeConfig` resource where the `Watchdog` metric is defined.

Reviewer: ```suggestion
. Remove the `Watchdog` alert metric by adding the `metricRelabelings` in the `platform-metrics-alerts` `ScrapeConfig` resource where the `ALERT` metric is defined.
```

//is there an exmaple of this? ^

> . To remove the `Watchdog` alert metric, add the `writeRelabelConfigs` specification to the `remoteWrite` configuration of the `PrometheusAgent` resource. The metric removal is complete after it is federated. Your `PrometheusAgent` resource might resemble the following YAML file:

Reviewer: I would use the ScrapeConfig rather. Here is an example:

```yaml
apiVersion: monitoring.rhobs/v1alpha1
kind: ScrapeConfig
metadata:
  name: platform-metrics-alerts
  namespace: open-cluster-management-observability
spec:
  jobName: alerts
  metricRelabelings:
    - action: labeldrop
      regex: managed_cluster|id
    - action: drop
      regex: ^Watchdog$
      sourceLabels:
        - alertname
  metricsPath: /federate
  params:
    'match[]':
      - '{__name__="ALERTS"}'
  scheme: HTTPS
  scrapeClass: not-configurable
  staticConfigs:
    - targets:
        - not-configurable
```

[#relabeling-mcoa-default-metrics]
= Relabeling default metrics for the multicluster observability add-on

Relabel configurations to modify or remove the default metrics that you are collecting. Relabel configurations in each `ScrapeConfig` resource or globally in the `remoteWrite` configuration of the `PrometheusAgent` resource.

*Required access:* Cluster administrator

.Prerequisites

- You have installed and enabled the multicluster observability add-on. For more information, see xref:../observability/obs_enable_mcoa.adoc#enable-mcoa[Enabling the multicluster observability add-on].

> 

Reviewer 2: ```suggestion

.Procedure

```
